firstvoices/backend/views/search/base_search_views.py

```diff
@@ -63,11 +56,11 @@ def get_search_params(self):
         Function to return search params in a structured format.
         """
         input_q = self.request.GET.get("q", "")
-        site_slug = self.request.GET.get("site_slug", "")
 
-        return {"q": input_q, "site_slug": site_slug}
+        search_params = {"q": input_q, "site_slug": ""}
```

Comment:
Minor: we could just leave out the site slug here instead of setting it to blank